src/implementations/twig/components/news-ticker/news-ticker.html.twig

```diff
@@ -71,21 +72,28 @@
 <div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
   {% if _items is not empty and _items is iterable %}
   <div class="ecl-news-ticker__container">
-    {% include '@ecl/icon/icon.html.twig' with {
-      icon: {
-        path: _icon_path,
-        name: 'information',
-        size: 'm',
-      },
-      extra_classes: 'ecl-news-ticker__icon',
-    } only %}
     <div class="ecl-news-ticker__content">
       <ul class="ecl-news-ticker__slides" id="{{ _aria_controls }}">
       {% for _item in _items %}
+        {% set _icon = '' %}
+        {% if _item.icon is not empty and _item.icon.name is not empty %}
+          {% set _icon %}
+            {% include '@ecl/icon/icon.html.twig' with {
+              icon: _item.icon|merge({
+                path: _icon_path,
+                name: _item.icon.name,
```

Comment:
can't we default the icon name to 'information' (at least until next major version)?
That way there won't be breaking changes here

Reply:
mmh..i'd say no, because we would alter the behavior of the component, currently you can set or not an icon for each item, if we default it to "something" it would always be there instead, and it would be totally arbitrary to show the info icon at this point. It would not work as backward compatibility, for that to happen we should check if in any of the item is present an icon and in case there is none we could "assume" that it's "old data" and apply this fallback but it seems an overhead for the template to do so.